crates/remora/src/bin/test_remora.rs

```diff
@@ -103,7 +106,7 @@ async fn deploy_testbed(tx_count: u64, duration: u64, pre_exec_workers: usize) -
     // Spawn txn generator.
     let configs = global_configs.clone();
     let id = 0;
-    let txn_generator = ExecutorShard::start(configs, id);
+    let _txn_generator = ExecutorShard::start(configs, id);
```

Comment:
This is a problem: it may get optimised away by the compiler and then the task panics. This happens roughly 10% of the time.